### PR TITLE
General cleanup

### PR DESCRIPTION
**Pull request by Yuhanun**
Did some general cleanup.

Removed a .h and .cpp file that were unused but I now realize they might just be for the future. If you want me to add it back i can do that real quick :)

### CODE REVIEW

**Review by HaicoDorenbos — COMMENTED** [review]
Cleaning up the code is always important to do :+1:

File: src/world/FieldComputations.cpp [hunk 1]
```diff
@@ -179,8 +175,8 @@ std::vector<LineSegment> FieldComputations::mergeBlockades(std::vector<LineSegme
     /* If two blockades intersect (in this case, overlap), we take the beginning of the first obstacle and the end of the second obstacle, and put them back in the front of the
      * obstacles vector. The second element gets erased. If they don't intersect, try the next two obstacles. Repeat this procedure until no overlaps are left. */
     std::sort(blockades.begin(), blockades.end(), [](const LineSegment &a, const LineSegment &b) { return a.start.y < b.start.y; });
-    int iterator = 0;
-    while (iterator < static_cast<int>(blockades.size()) - 1) {
```
Comment (HaicoDorenbos) [inline src/world/FieldComputations.cpp:183]: Be careful with these type of things. It often happens in the code that variables values get forgotten and for these situations a static_cast<int> is used.

File: src/stp/new_roles/PenaltyKeeper.cpp [hunk 2]
```diff
@@ -20,7 +20,7 @@ PenaltyKeeper::PenaltyKeeper(std::string name) : Role(std::move(name)) {
 Status PenaltyKeeper::update(StpInfo const& info) noexcept {
     // Failure if the required data is not present
     if (!info.getBall() || !info.getRobot() || !info.getField()) {
-        RTT_WARNING("Required information missing in the tactic info");
+        RTT_WARNING("Required information missing in the tactic info")
```
Comment (HaicoDorenbos) [inline src/stp/new_roles/PenaltyKeeper.cpp:23]: Huh? Every statement should be closed by a semicolon, right?

File: src/stp/new_roles/Keeper.cpp [hunk 3]
```diff
@@ -19,7 +19,7 @@ Keeper::Keeper(std::string name) : Role(std::move(name)) {
 Status Keeper::update(StpInfo const& info) noexcept {
     // Failure if the required data is not present
     if (!info.getBall() || !info.getRobot() || !info.getField()) {
-        RTT_WARNING("Required information missing in the tactic info");
+        RTT_WARNING("Required information missing in the tactic info")
```
Comment (HaicoDorenbos) [inline src/stp/new_roles/Keeper.cpp:22]: Why did you remove the semicolon?

**Review by HaicoDorenbos — CHANGES_REQUESTED** [review]
Semicolons are removed at some places, where I wonder whether they should be removed. Also make sure that the code still compiles and that all checks in Travis pass.

File: src/stp/new_roles/Keeper.cpp (hunk 3 above)
Comment (rolfvdhulst) [reply src/stp/new_roles/Keeper.cpp:22]: It's a macro

**Review by timovdk — CHANGES_REQUESTED** [review]
The implementations of the functions removed from ControlUtils.h are still in ControlUtils.cpp

File: src/world/FieldComputations.cpp (hunk 1 above)
Comment (Yuhanun) [reply src/world/FieldComputations.cpp:183]: Well static_cast<int>(size_t) can cause overflow.

Comment (Yuhanun) [reply src/world/FieldComputations.cpp:183]: Well static_cast<int>(size_t) can cause overflows so yeah

File: src/stp/new_roles/PenaltyKeeper.cpp (hunk 2 above)
Comment (Yuhanun) [reply src/stp/new_roles/PenaltyKeeper.cpp:23]: It's a macro defined to already contain the `;`

**Review by Yuhanun — COMMENTED** [review]
> The implementations of the functions removed from ControlUtils.h are still in ControlUtils.cpp

will fix that soonTM, been busy again sadly

**Review by HaicoDorenbos — APPROVED** [review]
Good job :+1:, removing and cleaning up code is often a very boring job, but really important since the code is more understandable and therefore reduces the risk of bugs, increases the efficiency of the code, decrease maintenance time, makes the system more suitable for evolution , etc. :+1:

However make sure to compile the code first before merging it and check if all tests in Travis pass.

**Comment by Yuhanun** [comment]
Issue it's having is because of roboteam_utils, either hasn't been merged yet or ye.

**Comment by Yuhanun** [comment]
95% confident this is because roboteam_utils is supposed to be on another branch btw ;)

**Review by timovdk — APPROVED** [review]
